Grand_Central_MIDI_Knobs/Grand_Central_MIDI_Knobs.py

```diff
@@ -0,0 +1,82 @@
+#  Grand Central MIDI Knobs
+#  for USB MIDI
+#  Reads analog inputs, sends out MIDI CC values
+#  written by John Park with Kattni Rambor and Jan Goolsby for range and hysteresis code
+
+import time
+import usb_midi
+import board
+from simpleio import map_range
+from analogio import AnalogIn
+print("---Grand Central MIDI Knobs---")
+
+midi_out = usb_midi.ports[1]  # Set the output MIDI channel (0-16)
+knob_count = 16  # Set the total number of potentiometers used
+
+# Knob list defines the characteristics of the potentiometers
+#  This list contains the input object, minimum value, and maximum value for each knob.
+#   example ranges:
+#   0 min, 127 max: full range control voltage
+#   36 (C2) min, 84 (B5) max: 49-note keyboard
+#   21 (A0) min, 108 (C8) max: 88-note grand piano
+#
+knb = [
+    (0, 36, 84),  # input 0: 36 (C2) min, 84 (B5) max: 49-note keyboard
+    (1, 36, 84),
+    (2, 36, 84),
+    (3, 36, 84),
+    (4, 36, 84),
+    (5, 36, 84),
+    (6, 36, 84),
+    (7, 36, 84),
+    (8, 0, 127),  # input 8: 0 min, 127 max: full range MIDI CC/control voltage for Rack
+    (9, 0, 127),
+    (10, 0, 127),
+    (11, 0, 127),
+    (12, 0, 127),
+    (13, 0, 127),
+    (14, 0, 127),
+    (15, 0, 127)
+]
+
+# Create the input objects list for potentiometers
+#  by getting pin # attributes using string formatting
+for k in range(knob_count):
+    knb[k] = (AnalogIn(getattr(board, "A{}".format(knb[k][0]))), knb[k][1], knb[k][2])
```

Comment:
dont re-assign knb[] - make a new array for the settings thats not shared

Reply:
OK, I'm on it.